### DISCUSSION TITLE
Unclear how to authenticate

### DISCUSSION THREAD

**Discussion by mklueh** [Intevel/nuxt-directus#51] [Q&A] [answered]
Hello,

So the documentation says

![image](https://user-images.githubusercontent.com/1553757/179414425-ea7850ac-0af3-482b-9688-0dec72324910.png)

However, I'm getting `403` errors, as I'm not authenticated and I have not seen anything clear in the docs on how to specify an access token to my directus backend. 

I generally only want to fetch content during the build phase to render static content. How would I do that?

**Comment by Intevel** [comment, marked as answer]
If your content is public, you don’t need an authentication, then you can set the View Permission of your collection in directus to public. 

If your content isn’t public you have to build your authentification.

**Reply by mklueh** [reply]
Hi @Intevel 

In my case the content should be private.

Could you elaborate what you mean with "you have to build your authentication"?
Is it enough to just call this method https://nuxt-directus.netlify.app/usage/usedirectusauth/
before hitting this endpoint https://nuxt-directus.netlify.app/usage/usedirectuscollections/ ?

**Reply by Intevel** [reply]
@mklueh The useDirectusAuth composable contains a login method, if you login with your directus user then you can access the collection/data.

**Reply by mklueh** [reply]
@Intevel Thanks, but can I also simply use the token I've created here in Directus or is there no way around using user username and password?

![image](https://user-images.githubusercontent.com/1553757/179910340-2b104926-72cf-4662-aa43-b90bb0c34553.png)

**Reply by craigharman** [reply]
Not currently, waiting on: https://github.com/Intevel/nuxt-directus/issues/31

**Reply by mklueh** [reply]
@craigharman Oh got it, thank you! Looking forward to it

**Reply by Intevel** [reply]
Yes, as @craigharman said. We are working on this.

**Reply by Intevel** [reply]
@mklueh Take a look at @mklueh, review it if you want. @craigharman you too please.

**Reply by mklueh** [reply]
@Intevel wow, thanks for the fast implementation! I'll try it out in the evening as soon as I have time

**Reply by Intevel** [reply]
@mklueh Can you please mark this discussion as answered?

**Reply by notflip** [reply]
How about files? I'm getting 403 errors with files only, do they have to be set to public somewhere?

**Reply by mklueh** [reply]
@notflip have you checked that the permissions within Directus itself are set correctly?

**Reply by notflip** [reply]
@mklueh Right, I set the Directus Files to All Access (for reading), is that correct?
